frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -586,6 +615,245 @@ export default function MotionToWaiveFees_Web({ data }) {
           dataKey="person.financialAssets"
           name="financialAssets"
         />
+        {data.person.financialAssets && (
+          <React.Fragment>
+            <CheckBox
+              label="Bank or credit union"
+              dataKey="person.financialBankOrCreditUnion"
+              name="bankOrCredit"
+            />
+            {data.person.financialBankOrCreditUnion && (
+              <TextInputGroup
+                groupLabel="Bank or credit union"
+                inputs={bankOrCredit}
+                buttonLabel="Bank or credit union"
+                maxInputs={2}
+                dataKey="person.bankOrCredit"
+              />
+            )}
+            <CheckBox
+              label="Stocks, bonds, securities, money market account, "
+              dataKey="person.financialStocksSecuritiesMoney"
+              name="stocksSecurities"
+            />
+            {data.person.financialStocksSecuritiesMoney && (
+              <div className="web-form-input">
+                <TextInput
+                  label="Account number"
+                  dataKey="person.stocksAccount"
+                />
+                <TextInput
+                  label="Date opened"
+                  dataKey="person.stocksDateOpened"
+                />
+                <TextInput
+                  label="Name of institution"
+                  dataKey="person.stocksName"
+                />
+                <TextInput
+                  label="Address of institution"
+                  dataKey="person.stocksAddress"
+                />
+                <TextInput
+                  label="Names on account"
+                  dataKey="person.stocksNames"
+                />
+                <TextInput
+                  label="Current Balance"
+                  dataKey="person.stocksBalance"
+                />
+              </div>
+            )}
+            <CheckBox
+              label="Retirement account"
+              dataKey="person.financialRetirementAccount"
+              name="retirementAccount"
+            />
+            {data.person.financialRetirementAccount && (
+              <div className="web-form-input">
+                <TextInput
+                  label="Account number"
+                  dataKey="person.retirementAccount"
+                />
+                <TextInput
+                  label="Date opened"
+                  dataKey="person.retirementDateOpened"
+                />
+                <TextInput
+                  label="Name of institution"
+                  dataKey="person.retirementName"
+                />
+                <TextInput
+                  label="Address of institution"
+                  dataKey="person.retirementAddress"
+                />
+                <TextInput
+                  label="Names on account"
+                  dataKey="person.retirementNames"
```

Comment:
The meaning of `person.retirementName` and `person.retirementNames` is not clear. Not only are their names similar, but also it doesn't communicate that this is related to a financial retirement account. I would prefer longer names that are more descriptive -- we are likely going to one day pay for how little we've thought through our data keys.